modules/mod_import_csv/templates/_admin_import.tpl

```diff
@@ -0,0 +1,15 @@
+{% extends "admin_base.tpl" %}
+
+{% block title %}{_ Import content _}{% endblock %}
+
+{% block content %}
+
+    <div class="admin-header">
+        <h2>{_ Import content _}</h2>
+    </div>
+
+    <div class="well">
+        {% all include "_admin_import_button.tpl" %}
```

Comment:
This is a way for other modules to add import buttons (e.g. Excel or WordPress import).